susemanager-utils/supportutils-plugin-susemanager/susemanager

```diff
@@ -21,18 +21,16 @@ addHeaderFile $OF
 
 log_cmd $OF "grep web.version /usr/share/rhn/config-defaults/rhn_web.conf"
 
-log_cmd $OF "hostname --fqdn"
+log_cmd $OF "echo $HOSTNAME"
```

Comment:
I'm not even sure we need this one as it gives the internal fqdn.

Reply:
yes, but we may see if the customer fiddle around with it